### DISCUSSION TITLE
`@bot.chat_member_handler` doesn't work sometimes.

### DISCUSSION THREAD

**Discussion by cc00010** [eternnoir/pyTelegramBotAPI#1919] [Q&A]
1. What version of pyTelegramBotAPI are you using?
`pip3 show pyTelegramBotApi`: `Version: 4.8.0`

3. What OS are you using?
MacOS

5. What version of python are you using?
`python --version`: `Python 3.9.7`

As the title says, `@bot.chat_member_handler` doesn't call the function sometimes. But it seems like after updating bot token it starts working as intended.

The code is:
```python3
@bot.chat_member_handler()
def chat_member_handler(status):
  print(status)

```

UPDATE: I already have `allowed_updates` set to `telebot.util.update_types`

**Comment by Badiboy** [comment]
chat_member update type is not in the default set of handled update types. Set it obviously when call palling/creating webhook

**Comment by cc00010** [comment]
@Badiboy Oh, I forgot to mention that I already have infinite polling set like this:
```python3
bot.infinity_polling(allowed_updates=telebot.util.update_types)
```

**Comment by cc00010** [comment]
and in webhook as well:
```python3
bot.set_webhook(url=webhook_url, drop_pending_updates=True, \
    allowed_updates=telebot.util.update_types)
```

**Comment by Badiboy** [comment]
If set - means that Telegram does not send them. Check bot settings (privacy etc.) and group settings (bot is admin, bot have enough permissions).

**Comment by cc00010** [comment]
Could you please tell me that particual setting? There are many of them.

**Comment by cc00010** [comment]
But the weird thing here is that if I revoke the token, the issue is gone.
